source/utilities/time/index.js

```diff
@@ -163,6 +163,21 @@ const formatDayMonthYearShort = (date = new Date(), locale='mx') => {
   return `${day} ${MONTHS_SHORT[locale][month-1]} ${year % 100}`;
 }
 
+/**
+ * Formats a date into a `MMM YY` readable string
+ *
+ * @example
+ * formatMonthYearShort(new Date(2018, 1, 2, 3, 4)); //=> 'Feb 18'
+ *
+ * @param {Date} [date] - Date. If not provided, `Date.now()`
+ * @param {string} [locale] - Locality string. `mx` by default
+ * @returns {string}
+ */
+const formatMonthYearShort = (date = new Date(), locale='mx') => {
+  let [year, month] = dateSpread(date, false);
+  return `${MONTHS_SHORT[locale][month-1]} ${year % 100}`;
+}
```

Comment:
Semicolon pana :)

Reply:
True that. I'll also include the semicolons for the rest of functions in this module.